contrib/reports/explore-server/src/client/util.tsx

```diff
@@ -239,9 +239,12 @@ export function getCve2tool2commitRuns(
   });
   return mapped;
 }
-export function countLocations(es: (Alert | Weakness)[]): number {
+export function countWeaknessLocations(es: Weakness[]): number {
   return new Set(es.map(e => `${e.file}:${e.line}`)).size;
 }
+export function countAlertLocations(es: Alert[]): number {
+  return new Set(es.flatMap(e => e.lines.map(line => `${e.file}:${line}`))).size;
+}
```

Comment:
Good point, thank you. I've updated the Alert counting part. I've also moved from `lines` field to `lineStart` and `lineEnd` fields, it is not neccessary to store the alerted lines line by line in case of a multi-line alert. Also, it's less heavy to check like this wheter a weakness line is within the range of alerted lines or not.